appctl_msg_defs/srv/Query.srv

```diff
@@ -0,0 +1,2 @@
+---
+string mode
```

Comment:
I see this and `Mode.msg` added, but I don't see them being removed from `appctl/`

Reply:
:skull: whoops.  Fixed :+1: :taco: